src/prefer-arrow-functions.spec.ts

```diff
@@ -7,6 +7,9 @@ import {
 } from './config';
 import rule from './prefer-arrow-functions';
 
+// TODO: Remove this once Jest starts supporting modern JavaScript
+globalThis.structuredClone ??= obj => JSON.parse(JSON.stringify(obj));
```

Comment:
Looks like ESLint now uses [structuredClone](https://mdn.io/structuredClone) so we need to "mock" it for Jest because Jest doesn't support that yet.